changelogs/unreleased/gh-10306-lua-call-runtime-access.md

```diff
@@ -0,0 +1,4 @@
+## feature/box
+
+* It is now possible to grant and revoke access to any `lua_call` function
+  in the internal API (gh-10306).
```

Comment:
I think we shouldn't advertize a use of an internal API anyhow, including information in the release notes. Let's drop the changelog entry.

Reply:
Changelog removed.